### DISCUSSION TITLE
Danalock State Stuck "Not Fully Locked"

### DISCUSSION THREAD

**Discussion by sembeeuk** [Koenkk/zigbee2mqtt#8786] [Devices]
The state of my Danalock V3-BTZB is stuck in "not_fully_locked". 

If I reset the lock (removing a battery), the state is initially correct, until I lock/unlock it for the first time, then it goes to that same state and doesn't change. A reconfiguration doesn't resolve the issue either. 

Despite the state being wrong, I can lock and unlock the lock, either using the button in the frontend or sending an MQTT command via Home Assistant. Unfortunately HA just stays on the state "lock", meaning any automations etc fail. 

Any suggestions?
Thanks.

**Comment by gfawkes66** [comment]
got to admit, i have 3 of these locks and find them a real challenge. Danalocks seem to be a constant challenge for status updates.
Usually, a reconfigure sorts this out for me but that procedure is giving me an error at the moment

you could try this but i would think about it first in case you give yourself more issues.
Hopefully, someone with a bit more experience may come up with a more 'Elegant' solution
I had this issue this morning and corrected it by :- 
First removing the 3 topics from the MQTT server that i could find (homassistant/sensor/deviceid, homassistant/lock/devicedid, sigbee2mqtt/device name)
Second, restart zigbee2mqtt so the topics are re - created.
third - i needed to do a reconfigure a while later, that seemed to fix it for me.
